### ISSUE TITLE
[TC-542] TO init.d script needs to be converted to LSB Init for Golang Proxy

### ISSUE THREAD

**Issue by limited** [apache/incubator-trafficcontrol#883]
None

Author: Dewayne Richardson
JIRA Link: <a href="https://issues.apache.org/jira/browse/TC-542">https://issues.apache.org/jira/browse/TC-542</a>
Found Version: 2.1.0

**Comment by dewrich** [comment]
```/etc/init.d/traffic_ops``` needs to be converted to the SystemD style of ```/etc/systemd/system/traffic_ops.service```

**Comment by rob05c** [comment]
Title is inaccurate. SystemD is not Linux Standard Base. SystemD is not a standard. Init scripts are the LSB standard. Converting our init scripts to SystemD is moving away from the standard.

**Comment by dewrich** [comment]
Still needs to be done

**Comment by ocket8888** [comment]
This doesn't actually affect the ability of a CDN to function, since right now all of our supported platforms support init.d scripts - so I'm marking this as minor.

**Comment by rob05c** [comment]
Yeah, I don't know that I agree it needs to be done. LSB init scripts work fine, and SystemD has no plans to stop supporting them AFAIK.

SystemD scripts can do some things LSB init can't (without external, UNIX-style coordination), and if we need those things we can do a rewrite. But if we don't, I don't think there's a reason to invest the development time to move away from the Standard.
